lib/iris/experimental/ugrid.py

```diff
@@ -1345,6 +1696,66 @@ def remove_connectivities(
         contains_edge=None,
         contains_face=None,
     ):
+        """
+        Remove one or more :class:`Connectivity` from the :class:`Mesh` that
+        match the provided criteria.
+
+        Criteria can be either specific properties or other objects with
+        metadata to be matched.
+
+        Kwargs:
```

Comment:
Need to remove the word `coordinate` by replicating the Kwarg descriptions you used for `Mesh.connectivities()` and `Mesh.connectivity()`.